### PR TITLE
Fix monorepo packages

### PR DESCRIPTION
**Pull request by cinaaaa**
## Description

I've just fixed the mismatched versions across the apps and packages but syncpack has a little issue with "workspace:*" versioning and detects it as mismatched [Link here](https://github.com/JamieMason/syncpack/issues/121).
but it's fine and I added a syncpackrc also for customizing the syncpack

From now the packages only should be installed with the recommended pnpm commands:
https://turbo.build/repo/docs/handbook/package-installation#addingremovingupgrading-packages

### CODE REVIEW

File: app/client-v2/apps/goat/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@p4b/goat",
-  "version": "1.0.0",
+  "version": "0.0.1",
```
Comment (JamieMason) [inline app/client-v2/apps/goat/package.json:3]: I think there is a bug in syncpack when workspace is enabled, this doesn't look right that it is changing the canonical package version. A fix will come in the next release.

Comment (cinaaaa) [reply app/client-v2/apps/goat/package.json:3]: Thanks @JamieMason

**Comment by majkshkurti** [comment]
Thanks @cinaaaa. Looks good. 
There is a conflict with pnpm-lock file. Can you fix and push ?

**Comment by cinaaaa** [comment]
@majkshkurti I've just temporarily flushed the lock file, just need a "pnpm i" after merging, It was not possible to resolve as there are three other file changes

**Pull request merged**